src/applications/mhv/landing-page/tests/e2e/restrict-by-facilities-list/facilities.content.cypress.spec.js

```diff
@@ -0,0 +1,108 @@
+import { appName } from '../../../manifest.json';
+import vamcEhr from '../../fixtures/vamc-ehr.json';
+import ApiInitializer from '../utilities/ApiInitializer';
+import LandingPage from '../pages/LandingPage';
+import { resolveLandingPageLinks } from '../../../utilities/data';
+
+const viewportSizes = ['va-top-desktop-1', 'va-top-mobile-1'];
+
+const noHealthDataHeading = /You do not have access to My HealtheVet tools/i;
+
+// Validate a card has a heading and the correct number of links in it.
+function validateLinkGroup(cardHeadline, numLinks) {
+  // Find the spotlight section and look for the links
+  cy.findByRole('heading', { name: cardHeadline }).should.exist;
+  cy.findByRole('heading', { name: cardHeadline })
+    .parents('[data-testid^="mhv-link-group"]')
```

Comment:
If we're not using a `data-testid` to select an element in the page, I don't know why we're adding that attribute to the HTML element.

Reply:
data-testid is only used for Cypress tests per VA documentation. I added these IDs to be able to search up the DOM to find the links within a card/hub group.

Reply:
Ok, I see. We may simplify these tests in the future. I can see it being helpful to assert the correct number of links in the unit test, and simply checking that the section is rendered in the e2e test. Fine for now. No change necessary.